01-getting-started/getting-started.Rmd

```diff
@@ -37,10 +46,10 @@ Each analysis is self-contained and provides information with how to obtain the
 We encourage you to download the [`.Rmd`](http://rmarkdown.rstudio.com) and follow the "getting started" section in the example before diving into our analysis examples.
 
 **Each analysis contains:**   
-- A README that introduces you to the analyses, concepts, requirements, and workflows for that module.
-- An R Notebook which consists of:
-     - An R markdown (`.Rmd`) file(s) that you can use in RStudio to run the analysis and contains it's own "getting started" section which describes how to download the example dataset from refine.bio. 
-     - An nb.html file that is the resulting output of the Rmd file rendered as an HTML file.
+- A README that introduces you to the analyses, concepts, requirements, and workflows for that module.  
+- An R Notebook which consists of:  
+    - An R markdown (`.Rmd`) file(s) that you can use in RStudio to run the analysis and contains it's own "getting started" section which describes how to download the example dataset from refine.bio.   
```

Comment:
These bullets are not indenting properly.

Reply:
I think this may be because you have extra spaces at the ends of the lines? Unless that was you trying to fix it?

Reply:
Yeah, I've added and deleted spaces.